client/keys/add.go

```diff
@@ -177,22 +177,45 @@ func runAddCmd(ctx client.Context, cmd *cobra.Command, args []string, inBuf *buf
 				return err
 			}
 
-			seenKeys := make(map[string]struct{})
-			for i, keyName := range multisigKeys {
-				if _, ok := seenKeys[keyName]; ok {
-					return fmt.Errorf("duplicate multisig keys: %s", keyName)
+			seenKeys := make(map[string]struct{})      // Tracks unique keys
+			duplicateKeys := make(map[string]struct{}) // Stores duplicate keys (only once)
+
+			for _, keyName := range multisigKeys {
+				if _, exists := seenKeys[keyName]; exists {
+					duplicateKeys[keyName] = struct{}{} // Store duplicate key once
+				} else {
+					seenKeys[keyName] = struct{}{} // Add new key to the set
 				}
-				seenKeys[keyName] = struct{}{}
+			}
+
+			if len(duplicateKeys) > 0 {
+				keysList := make([]string, 0, len(duplicateKeys))
+				for key := range duplicateKeys {
+					keysList = append(keysList, key) // Collect duplicate keys
+				}
+				sort.Strings(keysList) // Sort duplicates for consistent output
+
```

Comment:
```suggestion
				keysList := maps.Keys(duplicateKeys)
				slices.Sorted(keysList)
```
dont think we really need to sort it tbh. nbd

Reply:
Thanks for your suggestion! That looks much better.